policyengine_us/parameters/gov/states/ma/tax/income/credits/senior_circuit_breaker/eligibility/max_income.yaml

```diff
@@ -33,3 +38,5 @@ metadata:
       href: https://www.mass.gov/info-details/massachusetts-senior-circuit-breaker-tax-credit
     - title: 2022 Schedule CB
       href: https://www.mass.gov/doc/2022-schedule-cb-circuit-breaker-credit/download
+    - title: 2023 Schedule CB
+      href: https://www.mass.gov/doc/2023-schedule-cb-circuit-breaker-credit/download
```

Comment:
please also add the legal code or a comment to the legal code if it differs from the tax form amount